### PR TITLE
Warming up (dariodaic)

### PR DESCRIPTION
**Pull request by dai-ichy**
Hi Pat,

I'd like to join 4_adventure. I'm aware that I'm a little late to the party. I found out about it yesterday evening. You've stated in the project wiki that we must create separate pull requests for each warmup round, so this is the first one.

P.S. I'll send you an introductory email (so I don't spam the pull request comment).

D.

### CODE REVIEW

**Comment by patmaddox** [comment]
Go you!

File: 1_warmup/dariodaic/01_round/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+``` ruby
+puts "A journey of a thousand lines begines with a single Ruby step."
```
Comment (patmaddox) [inline 1_warmup/dariodaic/01_round/README.md:2]: true story!! <3

**Comment by dariodaic** [comment]
:) Playtime!

File: 1_warmup/dariodaic/01_round/README.md (hunk 1 above)
Comment (dzakman) [inline 1_warmup/dariodaic/01_round/README.md:2]: :) Playtime!

**Pull request merged**